### PR TITLE
Gamepage

### PR DESCRIPTION
**Pull request by Hanttone**
Please review and leave your comments! :) 
Built the navigation and added it to each page, also set up the project and added styleguidist to each component. 
The layout is only making sense in vertical mobile mode for now.

### CODE REVIEW

File: src/Assets/blueWave.svg [hunk 1]
```diff
@@ -0,0 +1,9 @@
+<<svg xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink" width="374.521" height="351.848" viewBox="0 0 374.521 351.848">
```
Comment (lmachens) [inline src/Assets/blueWave.svg:1]:
```suggestion
<svg xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink" width="374.521" height="351.848" viewBox="0 0 374.521 351.848">
```

File: src/Assets/greenFaded.svg [hunk 2]
```diff
@@ -0,0 +1,4 @@
+<svg xmlns="http://www.w3.org/2000/svg" width="375.041" height="185.43" viewBox="0 0 375.041 185.43">
+  <path id="Path_38" data-name="Path 38" d="M679.248,1235.562s86.553,3.181,149.016,31.724S910.078,1317.134,941.4,1346s36.658,43.84,67.955,47.866,44.936-17.586,44.936-17.586v44.714H679.248Z" transform="translate(-679.248 -1235.562)" fill="rgba(54,159,173,0.43)"/>
+</svg>
```
Comment (lmachens) [inline src/Assets/greenFaded.svg:3]: There are some not used properties in your SVGs. These properties are created by design tools like Adobe XD.
For example, you can remove `id` and `data-name`.
Take a look at https://www.svgminify.com/

File: src/Components/BottomLayout/BottomLayout.js [hunk 3]
```diff
@@ -0,0 +1,56 @@
...
+position: absolute;
+bottom: -1%;
+z-index: 1;
+`;
+
+const PurpleStrong = styled(Purple)`
+position: absolute;
+bottom: 0;
+z-index: 2;
+`;
```
Comment (lmachens) [inline src/Components/BottomLayout/BottomLayout.js:56]: You could simplify this, if you make use of the order in HTML.
I didn't test it, but give it a try :)

```suggestion
  svg {
    position: absolute;
    bottom: 0;
  }
  
  ${Lightpurple} {
    bottom: -1%;
  }
`;
```

**Review by lmachens — CHANGES_REQUESTED** [review]
Good job :).
It looks like your code formatter is not working. I would fix that first before merging it.

**Comment by Hanttone** [comment]
Prettier settings corrected and code improved.

**Pull request merged**